test/test-run-trace.js

```diff
@@ -6,15 +6,16 @@
 'use strict';
 
 var debug = require('./debug');
-var helper = require('./helper');
-var http = require('http');
-var run = helper.runWithControlChannel;
+var run = require('./run-with-ctl-channel');
 var tap = require('tap');
 
 var skipIfNoLicense = process.env.STRONGLOOP_LICENSE
                     ? false
```

Comment:
the code is just gone from test-run-metrics.js, and I haven't changed this, I could add a commit to change it, but it wouldn't behave any different, and the code it would be consistent with is now gone